x-pack/plugins/secops/public/components/formatted_ip/index.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { isArray, isEmpty, isString, uniq } from 'lodash/fp';
+import * as React from 'react';
+import { pure } from 'recompose';
+
+import { escapeQueryValue } from '../../lib/keury';
+import { DragEffects, DraggableWrapper } from '../drag_and_drop/draggable_wrapper';
+import { escapeDataProviderId } from '../drag_and_drop/helpers';
+import { getOrEmptyTagFromValue } from '../empty_value';
+import { IPDetailsLink } from '../links';
+import { parseQueryValue } from '../timeline/body/renderers';
+import { DataProvider } from '../timeline/data_providers/data_provider';
+import { Provider } from '../timeline/data_providers/provider';
+import { TruncatableText } from '../truncatable_text';
+
+const getUniqueId = ({
+  contextId,
+  eventId,
+  fieldName,
+  address,
+}: {
+  contextId: string;
+  eventId: string;
+  fieldName: string;
+  address: string | object | null | undefined;
+}) => `id-${contextId}-${fieldName}-${address}-for-event-${eventId}`;
```

Comment:
This might work out for you but you will get some weird outputs if address is actually a null or undefined.

You will get a `-null` or `-undefined`. Likewise if this is an object, did you want to maybe stringify it? Otherwise you will get `[object Object]` in most cases.

As a maintainer I would be suspicious of this not actually covering those use cases and maybe not be really unique.

I would handle those cases or at least use `tryStringify` below?